mod/message.php

```diff
@@ -282,7 +282,7 @@ function message_content(App $a)
 
 		$tpl = Renderer::getMarkupTemplate('msg-header.tpl');
 		DI::page()['htmlhead'] .= Renderer::replaceMacros($tpl, [
-			'$nickname' => $a->getLoggedInUserNickname(),
+			'$nickname' => DI::userSession()->getLocalUserNickname(),
```

Comment:
It looks like the `App $a` parameter of this function can be removed as these seem they were the only references to it.